setup.cfg

```diff
@@ -26,7 +26,7 @@ packages = find:
 python_requires = >=3.6
 install_requires =
   importlib_metadata
-  setuptools_scm
+  setuptools_scm>=4.1.0
```

Comment:
@chrisjbillington should this be here? I think it can be omitted since it is now specified in the build-system requirements of pyproject.toml. If it should stay, it then needs to be added to other projects (e.g labscript-devices) which do not have it in install_requires of setup.cfg.

Reply:
Yep! It still needs to be there.

It's both a runtime requirement (i.e. belongs in `install_requires`) since our  `__version__.py`s use it, and a build-time requirement (i.e. belongs in `pyproject.toml`) since our `setup.py`s use it.

Installing a pre-built wheel does not install its build-time requirements, and similarly, building a wheel does not install its run-time requirements. So it needs to be both in `pyproject.toml` and `setup_requires`.

`pyproject.toml` supersedes `setup_requires`, so nothing belongs in `setup_requires`.